src/restoreImpl.ts

```diff
@@ -31,11 +31,16 @@ async function restoreImpl(
         const cachePaths = utils.getInputAsArray(Inputs.Path, {
             required: true
         });
+        const enableCrossOsArchive = utils.getInputAsBool(
```

Comment:
Does `utils.getInputAsBool` assign the default value in case it is not specified by user. Please ensure you have tested the back compat scenario where existing workflows do not have this input set.

Reply:
I have added default in `action.yml` so that should work. Would test as well